rlclientlib/ranking_event.cc

```diff
@@ -33,17 +33,19 @@ namespace reinforcement_learning {
     }
     string context_str(context);
     copy(context_str.begin(), context_str.end(), std::back_inserter(_context));
+    _learning_mode = decision_mode;
   }
 
   const std::vector<unsigned char>& ranking_event::get_context() const { return _context; }
   const std::vector<uint64_t>& ranking_event::get_action_ids() const { return _action_ids_vector; }
   const std::vector<float>& ranking_event::get_probabilities() const { return _probilities_vector; }
   const std::string& ranking_event::get_model_id() const { return _model_id; }
   bool ranking_event::get_defered_action() const { return _deferred_action; }
+  const learning_mode ranking_event::get_learning_mode() const { return _learning_mode; }
```

Comment:
const is redundant here

Reply:
I mean the one associated with returned value

Reply:
Fixed.